diff --git a/play/src/pusher/controllers/FrontController.ts b/play/src/pusher/controllers/FrontController.ts
index dfdc55c..b6b5cfa 100644
--- a/play/src/pusher/controllers/FrontController.ts
+++ b/play/src/pusher/controllers/FrontController.ts
@@ -209,7 +209,7 @@ export class FrontController extends BaseHttpController {
             debug(`FrontController => [${req.method}] ${req.originalUrl} — IP: ${req.ip} — Time: ${Date.now()}`);
             res.json({
                 domain: process.env.PUSHER_URL,
-                name: process.env.SERVER_NAME || "WorkAdventure Server",
+                name: process.env.SERVER_NAME || "HOT-SPOT HQ",
                 motd: process.env.SERVER_MOTD || "A WorkAdventure Server",
                 icon: process.env.SERVER_ICON || process.env.PUSHER_URL + "/static/images/favicons/icon-512x512.png",
                 version: version + (process.env.NODE_ENV !== "production" ? "-dev" : ""),
diff --git a/play/src/pusher/services/LocalAdmin.ts b/play/src/pusher/services/LocalAdmin.ts
index 21bdc15..91b716f 100644
--- a/play/src/pusher/services/LocalAdmin.ts
+++ b/play/src/pusher/services/LocalAdmin.ts
@@ -310,7 +310,7 @@ class LocalAdmin implements AdminInterface {
             loadingLogo: null,
             loginSceneLogo: null,
             errorSceneLogo: null,
-            showPoweredBy: true,
+            showPoweredBy: false,
             loadingCowebsiteLogo: null,
             enableChat: ENABLE_CHAT,
             enableChatUpload: ENABLE_CHAT_UPLOAD,
diff --git a/play/src/pusher/services/MetaTagsBuilder.ts b/play/src/pusher/services/MetaTagsBuilder.ts
index 444b6f8..48c8491 100644
--- a/play/src/pusher/services/MetaTagsBuilder.ts
+++ b/play/src/pusher/services/MetaTagsBuilder.ts
@@ -13,11 +13,10 @@ import { ADMIN_API_URL } from "../enums/EnvironmentVariable";
 import { adminService } from "./AdminService";
 
 export const MetaTagsDefaultValue: RequiredMetaTagsData = {
-    title: "WorkAdventure",
-    description:
-        "Organize your online event in WorkAdventure to recreate spontaneous social interactions. Connect, collaborate and have fun with your teammates and attendees.",
-    author: "WorkAdventure team",
-    provider: "WorkAdventure",
+    title: "HOT-SPOT HQ",
+    description: "La oficina de HOT SPOT.",
+    author: "HOT SPOT",
+    provider: "HOT SPOT",
     favIcons: [
         {
             rel: "icon",
@@ -32,10 +31,10 @@ export const MetaTagsDefaultValue: RequiredMetaTagsData = {
             type: "image/png",
         },
     ],
-    appName: "WorkAdventure",
-    shortAppName: "WA",
-    themeColor: "#1B2A41",
-    cardImage: "https://workadventu.re/images/general/logo-og.png",
+    appName: "HOT-SPOT HQ",
+    shortAppName: "HQ",
+    themeColor: "#1C1715",
+    cardImage: "/static/images/favicons/icon-512x512.png",
 };
 
 export class MetaTagsBuilder {
